assets/src/edit-story/app/media/media3p/api/media3pApiProvider.js

```diff
@@ -67,9 +59,20 @@ function Media3pApiProvider({ children }) {
     selectedCategoryId,
     mediaType,
   }) {
-    if (provider.toLowerCase() !== Providers.UNSPLASH) {
+    if (
+      provider !== ProviderType.UNSPLASH &&
```

Comment:
Perhaps this could be:

if (!Object.keys(Providers).contains(provider)) {
  throw ...;
}

Reply:
Done.